### DISCUSSION TITLE
【灌水】看不到自己发过的贴子，悲(T_T)💔

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#735] [其他]
#599 #363 这是我发过的分享图片贴子

**Comment by trv3wood** [comment]
可以上Github Discussions页面找[Github Discussions](https://github.com/share121/inter-knot/discussions/)

**Reply by wugan-bot156** [reply]
我的意思是在绳网页面上找不到了/(ㄒoㄒ)/~~沉浸感没了

**Reply by lllliufeng** [reply]
> 我的意思是在绳网页面上找不到了/(ㄒoㄒ)/~~沉浸感没了

封坟了（确信

**Comment by lllliufeng** [comment]
现在可以翻到底了吧
